enable_ipu.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Constants
+venv_name=".graphium_ipu"
```

Comment:
Could we place this in a directory above the Graphium repo? I prefer to keep the ENVS separate from the working code, makes a bunch of checks / linting / searching etc easier